Client/mods/deathmatch/logic/rpc/CPlayerRPCs.cpp

```diff
@@ -168,3 +169,18 @@ void CPlayerRPCs::TakePlayerScreenShot(NetBitStreamInterface& bitStream)
 
     m_pClientGame->TakePlayerScreenShot(usSizeX, usSizeY, strTag, ucQuality, uiMaxBandwidth, usMaxPacketSize, pResource, uiServerSentTime);
 }
+
+void CPlayerRPCs::SetPlayerScriptDebugLevel(NetBitStreamInterface& stream)
+{
+    CClientPlayer* localPlayer = g_pClientGame->GetPlayerManager()->GetLocalPlayer();
+
+    if (!localPlayer)
+        return;
+
+    std::uint8_t scriptDebugLevel;
+
+    if (!stream.Read(scriptDebugLevel))
```

Comment:
A data type mismatch. You write uint32_t on server side, but here you have 8 bits read.

Reply:
Thx

Reply:
The better approach would be to replace all types with std::uint8_t, including the server-side one. But uint32_t is also okay.

Reply:
That's what I thought at first too, but I didn't want to change the old code, but it's ok, then make it eight bits everywhere.